lib/capybara/node/element.rb

```diff
@@ -462,11 +463,11 @@ def scroll_to(pos_or_el_or_x, y = nil, align: :top, offset: nil)
       ##
       #
       # Execute the given JS in the context of the element not returning a result. This is useful for scripts that return
-      # complex objects, such as jQuery statements. +execute_script+ should be used over
-      # +evaluate_script+ whenever possible. `this` in the script will refer to the element this is called on.
+      # complex objects, such as jQuery statements. {#execute_script} should be used over
+      # {#evaluate_script} whenever possible. `this` in the script will refer to the element this is called on.
```

Comment:
Please change "whenever possible" to "whenever a result is not expected or needed"